src/components/notifications/SMSPartner.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="mb-3">
+        <label for="smspartner-key" class="form-label">{{ $t("smspartnerApikey") }}</label>
+        <HiddenInput id="smspartner-key" v-model="$parent.notification.smspartnerApikey" :required="true" autocomplete="new-password"></HiddenInput>
+        <div class="form-text">
+            {{ $t("smspartnerApiurl") }}
+            <a href="https://my.smspartner.fr/dashboard/api" target="_blank">https://my.smspartner.fr/dashboard/api</a>
+        </div>
```

Comment:
Lets use component translation instead to allow for better translations.

```suggestion
        <i18n-t keypath="smspartnerApiurl" as="div" class="form-text">
            <a href="https://my.smspartner.fr/dashboard/api" target="_blank">https://my.smspartner.fr/dashboard/api</a>
        </i18n-t>
```